pkg/csi-common/utils.go

```diff
@@ -127,5 +128,5 @@ func logRedactedRequest(req interface{}) {
 		}
 	}
 	req1.Secrets = redactedSecrets
-	glog.V(5).Infof("GRPC request: %+v", req1)
+	log.Infof("GRPC request: %+v", req1)
```

Comment:
This should probably be debug level or lower.

Reply:
done!